tests/components/generic/conftest.py

```diff
@@ -29,3 +34,34 @@ def fakeimgbytes_svg():
         '<svg xmlns="http://www.w3.org/2000/svg"><circle r="50"/></svg>',
         encoding="utf-8",
     )
+
+
+@pytest.fixture
+def fakeimg_png(fakeimgbytes_png):
+    """Set up respx to respond to test url with fake image bytes."""
+    respx.get("http://127.0.0.1/testurl/1").respond(stream=fakeimgbytes_png)
+
+
+@pytest.fixture(scope="package")
+def mock_av_open():
+    """Fake container object with .streams.video[0] != None."""
+    fake = Mock()
+    fake.streams.video = ["fakevid"]
+    return patch(
+        "homeassistant.components.generic.config_flow.av.open",
+        return_value=fake,
+    )
+
+
+@pytest.fixture
+async def user_flow(hass):
+    """Initiate a user flow."""
+
+    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
We don't need to set up persistent_notification anymore.

Reply:
Thanks! done 7b9b92d